main.py

```diff
@@ -52,7 +52,7 @@
 
 executor = concurrent.futures.ThreadPoolExecutor(max_workers=20)
 
-cred = credentials.Certificate(eval(os.environ["FIREBASE_AUTH"]))
+cred = credentials.Certificate(os.environ["FIREBASE_AUTH"])
```

Comment:
I think we are keeping keeping eval as the auth credentials will be string. Would you be able to change this and not remove eval?
```suggestion
cred = credentials.Certificate(eval(os.environ["FIREBASE_AUTH"]))
```

Reply:
Ignore this PR, it was the first version and goes into the master branch of my forked repository, I will clean it up later. This is the link to the PR to the main repo: https://github.com/CoDaS-Lab/ASIST-API/pull/9

I have replaced `eval` there with `json.loads` as a safer option, let me know what you think!

Reply:
Merged the PR on the main repo. Ignore the suggested changes here as I think it won't be necessary as this was for reviewing changes. Thanks Anitej!